.circleci/config.yml

```diff
@@ -88,6 +88,7 @@ commands:
     steps:
       - run:
           name: Generate bytecode reports for the selected preset
+          no_output_timeout: 30m
```

Comment:
Unfortunately even after all that upfront work to get things working in parallel, it's pretty slow. Optimized via IR preset takes as long as 25 min on windows and on emscripten.

Reply:
Yeah. But the default timeout is 10 min so I'm increasing it, not decreasing it :)

I think 30 min is fine. If it fails we can still increase it but we'll get a clear signal that things are getting out of hand. 25 min is actually way more than I expected. I'd prefer this to finish much sooner. If not for the fact that we're working on IR performance already, I'd suggest parallelizing things even more to improve it.